data/strings/strings.txt

```diff
@@ -11748,6 +12009,7 @@
     ja = 目的地
     ko = 목적지
     lt = Maršrutas į
+    lv = Maršruts uz
```

Comment:
This might be too long for a small button. Maybe "Ceļš uz" and "Ceļš no"?

Reply:
Thanks, I like "Kurp", it sounds fresh!